ceno_zkvm/src/e2e.rs

```diff
@@ -231,6 +239,10 @@ impl<'a> ShardContext<'a> {
         let subcycle_per_insn = Tracer::SUBCYCLES_PER_INSN as usize;
         let max_threads = max_usable_threads();
 
+        let max_num_cross_shard_accesses = std::env::var("CENO_CROSS_SHARD_LIMIT")
```

Comment:
guess this parse from `env` design is some left over for, and later plan is to move it to command line argument ?